src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -166,6 +173,7 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
       }.bind(this), 500);
       shapefile.addTo(this.map);
     } else if (this.project) {
+      console.log('No shapefille!', this.shapefile[0]);
```

Comment:
is this a debugging item left in?

not sure on this project's policy of logging to console.

Reply:
Good call! I removed others but missed that one!

Reply:
Fixed in latest push.